### PR TITLE
AG-7089 Fixed click and highlight when tooltip off

### PR DESCRIPTION
**Pull request by alexanderby**
Fixed node click events and highlighting when tooltip is disabled.

### CODE REVIEW

**Review by alantreadway — COMMENTED** [review]
Code changes looks good, what testing strategy should we consider to avoid future regression?

File: charts-packages/ag-charts-community/src/chart/chart.test.ts [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+                xKey: dataSamples.randomAnnual.categoryKey,
+                yKey: dataSamples.randomAnnual.valueKey,
+            },
+            getNodeData: (series) => series.contextNodeData[0].nodeData,
+            getNodePoint: (item) => [item.point.x, item.point.y],
+            getDatumValues: (datum, series) => {
+                const xValue = datum[series['xKey']];
+                const yValue = datum[series['yKey']];
+                return [xValue, yValue];
+            },
```
Comment (alantreadway) [inline charts-packages/ag-charts-community/src/chart/chart.test.ts:204]: 🤔 Side-note: the fact we need these adaptor functions is probably pointing to the fact we should better standardise the `nodeData` format.

File: charts-packages/ag-charts-community/src/util/sanitize.ts [hunk 2]
```diff
@@ -5,6 +5,6 @@ export function sanitizeHtml(text?: string): string {
     if (!text) {
         return '';
     }
-    element.innerText = text;
+    element.textContent = text;
```
Comment (alantreadway) [inline charts-packages/ag-charts-community/src/util/sanitize.ts:8]: Does this still play well with HTML support for `series[].tooltip.renderer`?
https://ag-grid.com/javascript-charts/tooltips/#using-custom-tooltips

Comment (alantreadway) [reply charts-packages/ag-charts-community/src/util/sanitize.ts:8]: Tested locally, still looks good in Standalone Charts and Integrated Charts examples.

**Review by alantreadway — APPROVED** [review]
LGTM

File: charts-packages/ag-charts-community/src/chart/chart.test.ts (hunk 1 above)
Comment (alexanderby) [reply charts-packages/ag-charts-community/src/chart/chart.test.ts:204]: I agree, it's totally different everywhere

File: charts-packages/ag-charts-community/src/util/sanitize.ts (hunk 2 above)
Comment (alexanderby) [reply charts-packages/ag-charts-community/src/util/sanitize.ts:8]: Yes, but the custom HTML remains unsanitized. We only sanitise the fields and titles.

**Comment by alexanderby** [comment]
The tests for Pies and Histograms will be added later.

**Pull request merged**